includes/polling/os/hpe-rtups.inc.php

```diff
@@ -0,0 +1,8 @@
+<?php
+
+$version = trim(snmp_get($device, 'upsmgIdentFirmwareVersion.0', '-OQv', 'MG-SNMP-UPS-MIB'), '" ');
```

Comment:
Thanks for the PR.

Can you change all of the snmp_get() calls here to one snmp_get_multi_oid(). See includes/polling/os/extendair.inc.php as an example